pkg/check/smoke/load.go

```diff
@@ -102,24 +112,46 @@ func (c *LoadCheck) Run(ctx context.Context, cluster orchestration.Cluster, opts
 			txName := txName
 
 			go func() {
-				defer once.Do(func() { upload.Done() }) // don't wait for all uploads
+				defer once.Do(func() {
+					upload.Done()
+				}) // don't wait for all uploads
 				for retries := 10; txDuration == 0 && retries > 0; retries-- {
 					select {
 					case <-ctx.Done():
-						c.logger.Info("we are done")
+						c.log.Info("we are done")
 						return
 					default:
 					}
 
+					// Check the StorageRadius
+					for {
+						rs, err := clients[txName].ReserveState(ctx)
+						if err != nil {
+							c.log.Infof("error getting state: %v", err)
+							return
+						}
+						if rs.StorageRadius <= o.MaxStorageRadius {
+							break
+						}
+						c.log.Infof("waiting for StorageRadius to decrease. Current: %d, Max: %d", rs.StorageRadius, o.MaxStorageRadius)
+
+						select {
+						case <-ctx.Done():
+							c.log.Infof("context done in StorageRadius check: %v", ctx.Err())
+							return
+						case <-time.After(o.TxOnErrWait): // TODO: Adjust the sleep duration as needed
```

Comment:
What should be the adjustment here?

Reply:
Regarding context check, if global timeout happens, the context should be canceled, and we should stop execution.

Regarding the note: 
> **TODO:** Adjust the sleep duration as needed

I have already put a question in the PR description:

> Do we need a new config option for the sleep time, or can we utilize an existing one?

Reply:
this check does not need to happen so often, something like every 5 min, at the max.
let's add a new config option for this wait, with default being 5 min.